admin_scripts/ubuntu/ubuntu_16.04/layout/remove_bluetooth_applet.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    remove_bluetooth_applet.sh
+#%
+#% DESCRIPTION
+#%    This script removes the bluetooth applet if present.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         remove_bluetooth_applet (magenta.dk) 0.0.1
+#-    author          Danni Als
+#-    copyright       Copyright 2017, Magenta Aps"
+#-    license         GNU General Public License
+#-    email           danni@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2017/06/11 : danni : Script creation
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+# Remove Bluetooth indicator applet from Borger user
+BLUETOOTH_INDICATOR_PATH=$(find /usr/lib -name 'indicator-bluetooth-service')
+if [ ! -z "$BLUETOOTH_INDICATOR_PATH" ]
+then
+    chmod o-x $BLUETOOTH_INDICATOR_PATH
+fi
```

Comment:
Everything in `/usr/lib` is (or should be) owned by `root:root`, so this will also remove the applet for the superuser. Couldn't we just remove the `indicator-bluetooth` package?